.github/workflows/on_pull_request.yml

```diff
@@ -0,0 +1,52 @@
+name: On Pull Request (develop)
+
+on:
+  pull_request:
+    branches:
+      - develop
```

Comment:
As part of following the gitflow branching strategy, we may want to include main in here as well

Reply:
Yea, I was going to clarify that with Oli before I put it in